src/node_lttng_provider.h

```diff
@@ -61,7 +61,7 @@ void NODE_NET_STREAM_END(node_lttng_connection_t* conn,
 void NODE_GC_START(v8::GCType type,
                    v8::GCCallbackFlags flags,
                    v8::Isolate* isolate) {
-  const char* typeStr = "Unkown GC Type";
+  const char* typeStr = "Unkown GC Type"; // [sic]
```

Comment:
Well, there's been discussion of fixing this as a breaking change in the next major release. To unblock that part of this PR, I just noted it as a known typo so no one else tries to correct it.